### DISCUSSION TITLE
Request: Collaboration with the VIA software

### DISCUSSION THREAD

**Discussion by Plompi** [monkeytypegame/monkeytype#680] [Ideas]
it would be really great to have all we keyboard enthusiasts need in one software (VIA) and because a lot of people are using monkeytypes speedtest to determine their speed, it would be easier to have everything in one place. Woud love to see a collaboration between you.

**Comment by Miodec** [comment]
Im not exactly sure how a collaboration would work

**Reply by unrealapex** [reply]
Not sure how that would work too, but you're welcome to join the [Monkeytype discord server](https://www.discord.gg/monkeytype).
